lms/djangoapps/teams/models.py

```diff
@@ -128,15 +132,17 @@ class Meta(object):
     discussion_topic_id = models.SlugField(max_length=255, unique=True)
     name = models.CharField(max_length=255, db_index=True)
     course_id = CourseKeyField(max_length=255, db_index=True)
-    topic_id = models.CharField(max_length=255, db_index=True, blank=True)
+    topic_id = models.CharField(default='', max_length=255, db_index=True, blank=True)
```

Comment:
Should these be surfaced as with the time field above so we have defaults all defined the same way?

Reply:
Could you elaborate on that statement? I am not quite following.

Reply:
I think we was proposing that we also include default functions like:
```python
def default_string():
    return ''
```
...which isn't really necessary, since the `default` keyword is perfectly happy to take `''` as a value (although I understand Nathan's point regarding organization/consistency).

Reply:
The reason I used a callable instead of an inline default is because of the third answer for this [question](https://stackoverflow.com/questions/12649659/how-to-set-a-django-model-fields-default-value-to-a-function-call-callable-e)